specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/CommonDefinitions.json

```diff
@@ -1921,6 +1921,10 @@
           "description": "Version of Node.js.",
           "type": "string"
         },
+        "powerShellVersion": {
+          "description": "Version of PowerShell.",
+          "type": "string"
+        },
```

Comment:
New mutable properties need to go in a new api-version.  See breaking changes doc here:  [https://armwiki.azurewebsites.net/api_contracts/Intro.html](https://armwiki.azurewebsites.net/api_contracts/Intro.html)

Reply:
Hello @pilor, I had a quick sync with @ruslany, the owner of the swagger specs for our service. He said that we usually do not increase the API version for such as small change.

@ruslany : Could you please advice how to proceed with this? Thanks.

Reply:
The following description of the breaking change does not apply in this case because if the PUT request does not contain the new property then App service will preserve the current value of that property. Because of that we think there is no need to introduce new api version for this specific case.

New property added to response 

If a new property/field is added to the response an API, the GET-PUT pipeline will be broken. Consider the case where from portal a customer updates the value of a new property "A". Another customer does a GET of this resource using the SDK. The SDK will ignore the property since it does not understand it. From the SDK, the customer does a PUT using the model that was returned from the GET. This will overwrite the change made by the first customer from the portal.

Reply:
Thanks @ruslany for the followup.

Hi @pilor : Could you please approve this PR?